### PR TITLE
fix: update nhcarrigans avatar url

### PR DESCRIPTION
**Pull request by schmelto**

### CODE REVIEW

**Review by FayasNoushad — APPROVED** [review]

**Comment by FayasNoushad** [comment]
@schmelto you are a maintainer. So you can merge yourself. Or you can direct add a commit.

**Comment by FayasNoushad** [comment]
> @schmelto you are a maintainer. So you can merge yourself. Or you can direct add a commit.

Then why waiting?

**Review by Cahllagerfeld — APPROVED** [review]
LGTM 👍

File: public/data/_list.json
Comment (Cahllagerfeld) [inline public/data/_list.json:5]: I think the PR cant get merged, because the _list.json gets autogenerated now. The script reads the avatar-url from the corresponding json file in `public/data`

**Review by Cahllagerfeld — CHANGES_REQUESTED** [review]
I think these changes aren't needed anymore, due to autogeneration

We should have a look at the corresponding `nhcarrigan.json` file in the data-folder

File: public/data/_list.json
Comment (Cahllagerfeld) [reply public/data/_list.json:5]: It probably needs to be resolved in that file as well
[link](https://github.com/EddieHubCommunity/LinkFree/blob/352a98889ba2329eb1ee03927fb9e46f9a145ca7/public/data/nhcarrigan.json#L4)

**Review by Cahllagerfeld — APPROVED** [review]
awesome!

**Pull request merged**
